src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -44,12 +44,24 @@ public void storeMessage(Message message) {
 		Entity messageEntity = new Entity("Message", message.getId().toString());
 		messageEntity.setProperty("user", message.getUser());
 		messageEntity.setProperty("text", message.getText());
-		messageEntity.setProperty("timestamp", message.getTimestamp());
+		messageEntity.setProperty("timestamp", message.gettimestamp());
 		messageEntity.setProperty("imageUrl", message.getImageUrl());
+		messageEntity.setProperty("conversationTopic", message.getConversationTopicId());
```

Comment:
ConversationTopicId and imageUrl could be null, right ?

Reply:
yes they can. I was thinking on still saving them in the database as null and deal with this as I fetch messages in the front end. So I will have a something that checks for null imageUrls and conversationTopicIds. Does that work/make sense?